### ISSUE TITLE
No sound 

### ISSUE THREAD

**Issue by GeneralBelac** [mholtkamp/retro-league#6]
There is no sound for the cars driving also no music during the match

**Comment by mholtkamp** [comment]
Yes, there is no music when in a match. I think it would be nice to add some though. The cars should make a low pitch engine sound when you accelerate. So that is a bug if you don't hear that. What platform were you playing on by the way?
